app/src/main/java/com/khmelenko/lab/varis/log/FormattingOptions.java

```diff
@@ -0,0 +1,68 @@
+package com.khmelenko.lab.varis.log;
+
+public class FormattingOptions {
+    private Integer mTextColor;
+    private Integer mBackground;
+    private boolean mBold;
+    private boolean mItalic;
+    private boolean mUnderline;
+
+    public static FormattingOptions fromAnsiCodes(String[] ansiStates) {
+        FormattingOptions options = new FormattingOptions();
+        for (String ansiCode : ansiStates) {
+            AnsiCodes.applyAnsiCode(options, ansiCode);
+        }
+        return options;
+    }
+
+    public Integer getTextColor() {
+        return mTextColor;
+    }
+
+    public void setTextColor(Integer textColor) {
+        this.mTextColor = textColor;
```

Comment:
don't need to have this, sice we have `m` prefix for fields :)

Reply:
Of course 😄